README.md

```diff
@@ -60,16 +60,35 @@ and then clear those after the process starts.
 
 # Running with Docker
 
-Build the docker image:
+First copy the config file into a data directory and modify it to suit your needs:
 
 ```bash
-docker build . --tag bsk-subdomain-registrar
+mkdir -p data
+cp config-sample.json data/config.json
+vi config.json
```

Comment:
shouldn't this be `emacs config.json` ?

Reply:
I was going to put `nano` but that makes me not one of the cool kids 😜